packages/react/src/ImageSlider/ImageSlider.tsx

```diff
@@ -14,9 +14,9 @@ import { ImageSliderThumbnails } from './ImageSliderThumbnails'
 import { Ratio } from '../AspectRatio'
 import { Image, ImageProps } from '../Image/Image'
 
-export type ImageSliderImageProps = ImageProps & {
+export type ImageSliderImageProps = Omit<ImageProps, 'aspectRatio'> & {
   /** Specify the aspect ratio to use for the images. */
-  aspectRatio: Ratio
+  aspectRatio: Exclude<Ratio, 'tall' | 'x-tall' | '2x-wide'>
```

Comment:
I wonder if just ‘tall’ (4:3) images make the Image Slider difficult to use already. Let’s test if we can still support this aspect ratio.

Reply:
It's fine, and also the standard aspect for photography, so it makes sense to support it.